proto/wtools/abase/l0/l1/Countable.s

```diff
@@ -21,6 +21,7 @@ function is( src )
   if( _.primitive.is( src ) )
   return false;
 
+  // _.class.methodIteratorOf( src ) has already a check, asserting src is a routine; _.routineIs wrapper seems redundant.
```

Comment:
- _.class.methodIteratorOf( src ) has already a check, asserting src is a routine; _.routineIs wrapper seems redundant.